x-pack/winlogbeat/module/security/test/security_ingest_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// Windows is excluded not because the tests won't pass on Windows in general,
+// but because they won't pass on Windows in a VM — where we are using this — due
+// to the VM inception problem.
+//
+//go:build !windows
+// +build !windows
+
+package test
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/v7/x-pack/winlogbeat/module"
+)
+
+// Ignore these fields because they can be different on different versions
+// of windows.
+var ignoreFields = []string{
+	"event.ingested",
+	"message",
+
+	// This list is constructed through iteration over a Java hashmap
+	// and so is not consistently ordered.
+	"winlog.event_data.TicketOptionsDescription",
```

Comment:
The inconsistency in this could be avoided by using bit operations in the painless rather than the map iteration that is currently done.

I expect, but have not observed, that the same issue will be present in the `AccessMaskDescription` processing.